ADA/git_databricks.qmd

```diff
@@ -0,0 +1,231 @@
+---
+  title: "Databricks and version control"
+---
+
+<p class="text-muted">
+
+Guidance for analysts on how to connect Databricks to Github and Azure DevOps
+
+</p>
+
+------------------------------------------------------------------------
+
```

Comment:
Discussed this offline, but just adding here so there's a note on the PR...

I feel like we could put in a line or two at the beginning to say something along the lines of:

"For analytical projects developed on Databricks in the DfE, we recommend using git and Azure DevOps for sharing, collaborating and proper version control of work."